### ISSUE TITLE
Won't load.

### ISSUE THREAD

**Issue by AnonymousRonin** [rapid7/metasploit-framework#19178]
Seems as though the slowloris module doesn't load anymore.  I checked, there's actually tow files, one a .md file and the one in the module that should load is written in python (.py, not a .rb). I feel like rapid 7 does very little if any maintenance on the free version.  Most of the exploit require extensive mods to work, it never uses the assigned lport and it returns strange errors when run against target that have already proven exploitable.  I noticed the slowloris code looks a little wonky too, it probably doesn't work out of the box.  I think it's safe to say msfconsole has fallen into the category of irrelavent.  Especially with tools like burp suite for $500 a year versus the thousands rapid 7 charges.  Btw, how come you never load any exploits that actually worked?

**Comment by bcoles** [comment]
> the module that should load is written in python

The Python modules are generally less used and less maintained. It is entirely possible the module has a bug.

> it returns strange errors

Spooky. It is a mystery. I guess we'll never know.

> Especially with tools like burp suite for $500 a year versus the thousands rapid 7 charges.

Different tools. Different purpose. Irrelevant comparison. Metasploit is free.

**Comment by adfoster-r7** [comment]
The module works for me:

```
msf6 auxiliary(dos/http/slowloris) > set rhost 127.0.0.1
rhost => 127.0.0.1
msf6 auxiliary(dos/http/slowloris) > set rport 8000
rport => 8000
msf6 auxiliary(dos/http/slowloris) > run

[*] Starting server...
[*] Attacking 127.0.0.1 with 150 sockets
[*] Creating sockets...
[*] Creating socket number 0
[*] Creating socket number 1
...
[*] Creating socket number 150
[*] Sending keep-alive headers... Socket count: 150
```

Maybe you've got an issue with your specific python configuration, or maybe I don't understand the issue; but without the bug template being filled out - there's not enough detail to investigate further

**Comment by nrathaus** [comment]
Works for me as well.. I am using default setup of metasploit on Debian 12

**Comment by AnonymousRonin** [comment]
Yeah, I'm using a brand new install of Kali. But I actually found that particular exploit works better being deployed outside of metasploit so it's a mute point now.

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by AnonymousRonin** [comment]
Like I said, the module won't even load. It's a brand new install of kali. I'm gonna spin up a vm and see what happens.

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by nrathaus** [comment]
I booted a fresh Kali (a VDI image from their site) under VirtualBox and had no issue calling
`use dos/http/slowloris`

It loads and I can use it
